css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-details {
+		display: flex;
+		flex-grow: 1;
+		padding-block: $hero-padding-xs;
+
+		@include tablet {
+			justify-content: flex-end;
+			padding-block: $hero-padding-sm;
+			padding-inline-start: $hero-padding-xs;
+		}
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-start: $hero-padding-sm;
+			padding-inline-end: $hero-padding-sm;
+		}
+
+		.hero-details-card {
+			align-self: flex-start;
+			backdrop-filter: blur(20px);
+
+			.theme-high-contrast & {
+				background: unset;
+				background-color: var(--theme-body-background);
+				opacity: 1;
+			}
+
+			@include desktop {
+				background: linear-gradient(
```

Comment:
Pertaining to this linear-gradient

1. Why only on desktop? This ends up looking a bit odd on mobile.
1. A big part of me thinks the right thing for this card is to have it as its own component ... thoughts?
1. Sass variable are preferable so that if we misspell them or use one that's removed we get a build error. The interpolation syntax is awkward: `#{$variable}`; So this will become:
   ```scss
   background: linear-gradient(
		 116deg,
			#{$body-background} 23.94%,
			#{$border-white-high-contrast} 200%
		);
   ```

Reply:
1. Mobile hides the background so fancy BG becomes superfluous
2. I dont hate the idea.   Happy to refactor if thats the case.
3. Ill adjust.

Reply:
@wibjorn  adjusted to your notes.

Reply:
The issue with 1 is that it seems unexpected to me and might produce color contrast issues on mobile that would required extra work from us - it would all depend on the background. It is not a given that all heros always have the same background color.

Reply:
I feel like that would be a good thing.  To render it on mobile, but that would require diligence with image size and placement.  do we want to require devs/designers adding heros to account for this?

Reply:
I'm not sure I follow. I think it's pretty safe to make the card the same on all screen sizes. In fact, I think it's the safest option in terms of avoiding contrast issues.